crates/rpc/rpc/src/eth/revm_utils.rs

```diff
@@ -365,20 +370,30 @@ impl CallFees {
     ///
     /// If no `gasPrice` or `maxFeePerGas` is set, then the `gas_price` in the returned `gas_price`
     /// will be `0`. See: <https://github.com/ethereum/go-ethereum/blob/2754b197c935ee63101cbbca2752338246384fec/internal/ethapi/transaction_args.go#L242-L255>
+    ///
+    /// if 'gasPrice' is None , 'call_max_fee', 'call_priority_fee','max_fee_per_blob_gas' is set,
+    /// we treat it as 4844 transaction.
+    /// Revert if transaction is 4844 and 'blob_versioned_hashes' is None
```

Comment:
there is nothing to revert here.

please use different terminology than revert